accessories/switch.js

```diff
@@ -59,7 +59,7 @@ HomeAssistantSwitch.prototype = {
       this.log(`Setting power state on the '${this.name}' to on`);
 
       this.client.callService(callDomain, 'turn_on', serviceData, (data) => {
-        if (this.domain === 'scene') {
+        if (this.domain === 'scene' || this.domain === 'script') {
```

Comment:
Replace line 62 with
`if (this.domain === 'scene' || (this.domain === 'script' && !(this.data.attributes.can_cancel))) {`

Then, replace line 171 with
`if (this.domain === 'remote' || this.domain === 'switch' || this.domain === 'input_boolean' || this.domain === 'group' || this.domain === 'automation' || (this.domain === 'script' && this.data.attributes.can_cancel)) {`

Reply:
Also, please update the README to explain script support

I created a branch for script support to test my comments above, you can refer to it if you want (https://github.com/schmittx/homebridge-homeassistant/tree/script-support)